backend/src/services/box/database/operations.py

```
@@ -0,0 +1,1442 @@
+            )
+            .scalars()
+            .first()
+        )
+
+        if existing:
+            raise conflict_error(
+                "Item with the same name already exists",
+                conflicts=[
+                    {"type": "folder", "id": existing.id, "name": existing.name}
+                ],
+            )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)